ed_sensor_integration/tools/fitter_viz_data.cpp

```diff
@@ -0,0 +1,117 @@
+#include <ed/entity.h>
+#include <ed/world_model.h>
+
+#include <ed/kinect/fitter.h>
+
+#include <geolib/math_types.h>
+
+#include <opencv2/highgui/highgui.hpp>
+
+#include "ed_sensor_integration/tools/fitter_viz.h"
+#include "ed_sensor_integration/tools/snapshot.h"
+
+
+/**
+ * @brief usage, print how the executable should be used and explain the input
+ */
+void usage()
+{
+    std::cout << "Usage: ed_fitter IMAGE-FILE-OR-DIRECTORY  WORLDMODEL_NAME  ENTITY_ID" << std::endl;
+}
+
+/**
+ * @brief main executable to visualise the fitting process of stored images.
+ * @param argc: should be 4
+ * @return
+ */
+int main(int argc, char **argv)
+{
+    if (argc != 4)
+    {
+        usage();
+        return 1;
+    }
+
+    std::string model_name = argv[2];
+
+    ed::WorldModelPtr world_model;
+    try
+    {
+        world_model = ed::loadWorldModel(model_name);
+    }
+    catch (ed::ModelNotFoundException e)
+    {
+        std::cerr << "World model '" << model_name << "' could not be loaded." << std::endl;
+        std::cerr << e.what() << std::endl;
+        return 1;
+    }
+
+    std::string entity_id = argv[3];
+    ed::EntityConstPtr e = world_model->getEntity(entity_id);
+    if (!e)
+    {
+        std::cerr << "Entity '" << entity_id << "' could not be found in world model '" << model_name << "'." << std::endl;
+        return 1;
+    }
+
+    tue::filesystem::Path path = argv[1];
+    if (!path.exists())
+    {
+        std::cerr << "Path '" << path << "' does not exist." << std::endl;
+        return 1;
+    }
+
+    ed::SnapshotCrawler crawler(path);
+
+    Fitter fitter;
+
+    while(true)
```

Comment:
Also add ros ok check here

Reply:
The data visualization does not use ros at all. So I kept ros ok out of it here

Reply:
Check